### DISCUSSION TITLE
How to show individual boxes for each running pod

### DISCUSSION THREAD

**Discussion by emmahsax** [argoproj/argo-cd#18238] [Q&A] [answered]
I have multiple instances of ArgoCD running, all of them on the same versions:
```json
{
    "Version": "v2.10.9+c071af8",
    "BuildDate": "2024-04-30T15:53:28Z",
    "GitCommit": "c071af808170bfc39cbdf6b9be4d0212dd66db0c",
    "GitTreeState": "clean",
    "GoVersion": "go1.21.3",
    "Compiler": "gc",
    "Platform": "linux/amd64",
    "KustomizeVersion": "v5.2.1 2023-10-19T20:13:51Z",
    "HelmVersion": "v3.14.3+gf03cc04",
    "KubectlVersion": "v0.26.11",
    "JsonnetVersion": "v0.20.0"
}
```

However, on one of them, the pods show up like this, with different little starred boxes for each pod:

![image](https://github.com/argoproj/argo-cd/assets/7562793/413ec0ec-dfea-4637-85cc-0917289aff56)

But on the other instance, the pods show up like this, with English text indicating how many pods there are:

![image](https://github.com/argoproj/argo-cd/assets/7562793/57cc7241-d1ad-451b-93b2-c2b0c01eb9e7)

I'd love for all of them to show up with the little boxes. I'm also not sure what the star indicates, too, so if anybody has any guidance of how to tweak those settings or what the star means, that'd be greatly appreciated!

**Comment by ChristianCiach** [comment]
The stars indicate that the pods are less than 30 minutes old. See https://github.com/argoproj/argo-cd/pull/8006

**Comment by ChristianCiach** [comment, marked as answer]
The grouping of the individual boxes into a single `Healthy x pods` group happens if the application contains more than 15 pods in total. This has been introduced in https://github.com/argoproj/argo-cd/pull/12089

Looking at the code, this should be configurable, but I am not sure how exactly. I am on mobile and browsing the code is hard, but I will come back to you tomorrow if nobody else provides an answer until then.

**Reply by ChristianCiach** [reply]
Unfortunately, this is not configurable. At least not without editing the local storage of your browser where the preferences are saved as JSON. ArgoCD does not provide a way to edit this specific value.

**Reply by emmahsax** [reply]
Perfect description, thank you so much! Since it's not editable, it's not a huge deal... it is what it is.
